test/e2e/e2e.yaml

```diff
@@ -0,0 +1,117 @@
+      command: |
+        kubectl label namespace skywalking-system swck-injection=enabled
+        kubectl apply -f test/e2e/demo.yaml
+      wait:
+        - namespace: skywalking-system
+          resource: deployment/demo
+          for: condition=Available 
+  kind:
+    import-images:
+      - quay.io/jetstack/cert-manager-cainjector:v1.3.1
+      - quay.io/jetstack/cert-manager-controller:v1.3.1
+      - quay.io/jetstack/cert-manager-webhook:v1.3.1
+      - docker.io/kubesphere/kube-rbac-proxy:v0.4.1
+      - docker.io/apache/skywalking-oap-server:8.8.1
+      - docker.io/apache/skywalking-ui:8.8.1
+      - ghcr.io/apache/skywalking-swck-spring-demo:v0.0.1
+      - docker.io/apache/skywalking-java-agent:8.7.0-jdk8
+    expose-ports:
+      - namespace: skywalking-system
+        resource: service/demo 
+        port: 8085
+      - namespace: skywalking-system
+        resource: service/skywalking-system-oap
+        port: 12800
+      - namespace: skywalking-system
+        resource: service/skywalking-system-ui
+        port: 80
+  timeout: 20m
+
+cleanup:
+  # always never success failure
+  on: always
+
+trigger:
+  action: http
+  interval: 30s
+  times: 5
+  url: http://${service_demo_host}:${service_demo_8085}/hello
+  method: GET
+
+verify:
+  # verify with retry strategy
+  retry:
+    # max retry count
+    count: 10
+    # the interval between two attempts, e.g. 10s, 1m.
+    interval: 10s
+  cases:
+    - query: swctl --display yaml --base-url=http://${service_skywalking_system_oap_host}:${service_skywalking_system_oap_12800}/graphql service ls
+      expected: verify/service.yaml
+    - query: swctl --display yaml --base-url=http://${service_skywalking_system_oap_host}:${service_skywalking_system_oap_12800}/graphql metrics linear --name service_cpm --service-name Your_ApplicationName | yq e 'to_entries' -
+      expected: verify/metrics.yaml
+    - query: swctl --display yaml --base-url=http://${service_skywalking_system_oap_host}:${service_skywalking_system_oap_12800}/graphql endpoint list --keyword=hello --service-name Your_ApplicationName
+      expected: verify/endpoint.yaml
+    - query: swctl --display yaml --base-url=http://${service_skywalking_system_oap_host}:${service_skywalking_system_oap_12800}/graphql metrics linear --name endpoint_cpm --endpoint-name {GET}/hello --service-name Your_ApplicationName | yq e 'to_entries' -
+      expected: verify/metrics.yaml
+    - query: 'curl -s \
+            --request POST \
+            --header "Content-Type: application/json;charset=UTF-8" \
+            --data ''{"query":"query queryEndpoints($serviceId: ID!, $keyword: String!) {\n    getEndpoints: searchEndpoint(serviceId: $serviceId, keyword: $keyword, limit: 100) {\n      key: id\n      label: name\n    }\n}","variables":{serviceId: "WW91cl9BcHBsaWNhdGlvbk5hbWU=.1", keyword: ""}}'' \
+            http://${service_skywalking_system_ui_host}:${service_skywalking_system_ui_80} 2>&1 | yq e ".data.getEndpoints" -P -'
```

Comment:
Why do you use a raw `curl` to query, doesn't `swctl endpoint list ......` help?

Reply:
I want to simulate the UI request to test whether the UI is normal.

Reply:
you can use `swctl endpoint list` query `${service_skywalking_system_ui_host}:${service_skywalking_system_ui_80}` directly.

Reply:
Make sense to me, thanks a lot.